### DISCUSSION TITLE
@nomicfoundation/hardhat-toolbox require

### DISCUSSION THREAD

**Discussion by 6be709c0** [NomicFoundation/hardhat#4002] [General]
Hello,

I'm reading through the guide and there's multiple parts in the repo mentioning using 
```
const {
  loadFixture,
} = require("@nomicfoundation/hardhat-toolbox/network-helpers");
```
I'm getting the error `Error: Cannot find module '@nomicfoundation/hardhat-toolbox/network-helpers'`
The latest version 2.0.2 deployed 3months ago works with 
```
const {
  loadFixture,
} = require("@nomicfoundation/hardhat-network-helpers");
```

I'm seeing modification on those files few hours ago, is it because a new version is being released or am I missing something?

**Comment by 0xMouiz** [comment]
I am using `version 3.0.0` and getting same issue whenever I run `npx hardhat test`, the problem is with the import but no idea how to solve it

**Reply by fvictorio** [reply]
Happy to look into this if any of you can provide minimal reproduction steps or a link to a repo.
